Shared/Sources/ItemSection.swift

```diff
@@ -0,0 +1,8 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import Foundation
+
+struct ItemSection: RevertSection {
```

Comment:
I don't quite get the name `ItemSection`.  Is it a generic `Section` or only can be used in `AlertViewController`?

Reply:
This struct is meant to represent Section of `Item` item. Yes, we do have an item model named `Item`.
The naming of `ItemSection` is just following the current style (`HomeItem`, `ColorItem`,  etc).

This section is actually being used in multiple view controllers such as `ControlsViewController`.

Reply:
Actually, correction, the section is only being used in `ControlsViewController`, but `Item` is being used in bot `AlertViewController` and `ControlsViewController`.

Shoiuld I change this to `ControlsSection` instead?

Reply:
If it's from old codebase we can just skip it. 😄